mzLib/Test/Test.csproj

```diff
@@ -132,6 +133,9 @@
     <None Update="DatabaseTests\ProteoformClassificationUnitTest.csv">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="DatabaseTests\PSI-MOD.obo">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
We've usually used "Always;" is this intentional?

Reply:
I did it because the PSI-MOD.obo file shouldn't need to change unless there's a new version from the mzml people. I suppose I could make it dynamically fetch it from GitHub, but I don't have time to write that method.